suite-native/transactions/src/components/TransactionsEmptyState.tsx

```diff
@@ -7,43 +7,49 @@ import {
     RootStackRoutes,
     StackNavigationProps,
 } from '@suite-native/navigation';
+import { Translation, useTranslate } from '@suite-native/intl';
+
+const wrapperStyle = prepareNativeStyle(() => ({
+    paddingHorizontal: 0,
+}));
 
 const cardStyle = prepareNativeStyle(utils => ({
-    justifyContent: 'center',
     alignItems: 'center',
     flex: 1,
     width: '100%',
     paddingHorizontal: utils.spacings.large,
+    paddingVertical: utils.spacings.extraLarge,
     borderRadius: utils.borders.radii.large,
 }));
 
 const receiveButtonStyle = prepareNativeStyle(() => ({
-    width: '90%',
+    width: '100%',
 }));
 
 export const TransactionsEmptyState = ({ accountKey }: { accountKey: string }) => {
     const navigation =
         useNavigation<StackNavigationProps<RootStackParamList, RootStackRoutes.ReceiveModal>>();
     const { applyStyle } = useNativeStyles();
+    const { translate } = useTranslate();
 
     const handleReceive = () => {
         navigation.navigate(RootStackRoutes.ReceiveModal, { accountKey });
     };
 
     return (
-        <Box paddingHorizontal="medium">
+        <Box style={applyStyle(wrapperStyle)}>
             <Card style={applyStyle(cardStyle)}>
                 <Box marginBottom="large" alignItems="center">
                     <Pictogram
                         variant="green"
                         icon="stack"
-                        title="No transactions"
-                        subtitle="Get started by receiving coins"
+                        title={<Translation id="transactions.emptyState.title" />}
```

Comment:
why is one of these props uses`Translation` component, but other uses `translate()` callback? Unify this please

Reply:
Because `<Pictogram />` allows `ReactNode` for title, but asks  for `string | undefined` for description

Reply:
Better to change the type of Pictogram subtitle...

Reply:
bb6bfa6f66